src/main/java/dev/andrylat/vomelianchuk/finances/mortgagecalculator/consoleio/MortgageCalculatorConsoleIO.java

```diff
@@ -0,0 +1,58 @@
+package dev.andrylat.vomelianchuk.finances.mortgagecalculator.consoleio;
+
+import dev.andrylat.vomelianchuk.finances.common.ConsoleIO;
+import dev.andrylat.vomelianchuk.finances.mortgagecalculator.exceptions.DataMortgageCalculatorException;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+
+public class MortgageCalculatorConsoleIO implements ConsoleIO<List<Double>> {
+    private static final String TEXT_MONTHLY_PAYMENT =  "\nMonthly payment = ";
+    private static final String ERROR_BEGIN =  "\nERROR: ";
+    private static final String INCORRECT_DATA =  "Data must be numbers";
+
+    private final String[] promptsInput = {"Input total price:", "Input down payment:", "Input  interest rate:", "Input number of years:"};
+
+    private static InputStream input;
```

Comment:
WHY do you use static context?! You create OBJECT but fields are static. If you create two objects of this class, one will use system out and in, but second - file, everything will be in the FILE! because these fields ARE COMMON for both objects.

Reply:
Fixed here: d7b1b80